### DISCUSSION TITLE
There are triggers for start of the session?

### DISCUSSION THREAD

**Discussion by supremeremixx** [br-g/openf1#93] [Q&A]
Hello!
I would like to get data from live session but minimize the amount of requests to the API. To do this, I need to start to send requests only after a new session has started. There are any triggers/flags that would signal the start of a new session?

**Comment by redvoron** [comment]
No, there are no triggers. I compare the current time with the session start time and then start sending requests to the API.

**Reply by supremeremixx** [reply]
Yep, I also considered such approach, but what if some session will be cancelled, rescheduled or (like FP2 of Las Vegas 23) will be longer than expected? As I understand Start and End time from API don't dynamically change for such situations.
I've been thinking about cases like this and that's why I tried to found out about any triggers or flags.
